### PR TITLE
feat: Add version support and release pipeline

### PR DESCRIPTION
**Pull request by mbevc1**
*Issue #13 
Fixes: #13 

*Description of changes:*
* Adds support for binary version
* Add automatic release pipeline via GoReleaser by pushing a new tag
* New Makefile targets for local execution and build
* Update README to reflect the changes
* Removal of empty spaces and formatting

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by tzneal** [comment]
Yes, sorry, I meant to comment on this.  I need to get an internal approval before beginning to publish binaries, but once that is complete I'm happy to accept.

**Comment by mbevc1** [comment]
No worries, thanks!

**Comment by mbevc1** [comment]
Is there any movement on this @tzneal , I've noticed there was an initial release last week :tada:

**Comment by tzneal** [comment]
Hey @mbevc1, it's approved now so this can go in. Can you rebase this PR? or I can do it if you don't have time.

**Comment by mbevc1** [comment]
Sure, I'll rebase this later today and we can get it merged. Cheers

File: README.md [hunk 1]
```diff
@@ -14,6 +15,7 @@
 
 ### Installation
 
+Please either fetch th latest [release](https://github.com/awslabs/eks-node-viewer/releases) or install manually using:
```
Comment (tzneal) [inline README.md:18]:
```suggestion
Please either fetch the latest [release](https://github.com/awslabs/eks-node-viewer/releases) or install manually using:
```

**Review by tzneal — APPROVED** [review]
lgtm, thanks!

**Pull request merged**